packages/swap/src/routes/quote.ts

```diff
@@ -226,7 +226,9 @@ const quote = (io: Server) => {
         const message =
           err instanceof Error ? err.message : 'unknown error (possibly no liquidity)';
 
-        logger.error('error while collecting quotes:', err);
+        if (!message.includes('InsufficientLiquidity')) {
+          logger.error('error while collecting quotes:', err);
+        }
```

Comment:
maybe as a warning and a have a seperate alert to warn JIT in case it goes over our threshold?